### ISSUE TITLE
Admin Tab Text Invisible

### ISSUE THREAD

**Issue by brchristian** [solidusio/solidus#3347]
I just made a brand new Solidus store with seed data using the steps outlined in the README.

Admin tab text is invisible (the same color as the background). I assume this it not intended!

**Solidus Version**:
2.9.1

**To Reproduce**

Create default Solidus store.
1. Go to http://localhost:3000/admin/products/ruby-baseball-jersey/edit
-->

**Current behavior**
Admin tabs are invisible / not readable.

**Expected behavior**
Admin tab text should be legible.

**Screenshots**
<img width="1300" alt="Screen Shot 2019-09-24 at 3 27 36 PM" src="https://user-images.githubusercontent.com/2460418/65554932-d8547e00-dedf-11e9-98b6-80ffd35aa83c.png">

<img width="369" alt="Screen Shot 2019-09-24 at 3 28 37 PM" src="https://user-images.githubusercontent.com/2460418/65554972-fc17c400-dedf-11e9-9e4e-7c2e0b9ac35f.png">

**Desktop (please complete the following information):**
 - OS: MacOS 10.14.6
 - Browser: Safari
 - Version: 12.1.2

**Additional context**

My Gemfile is below, for reference:

````
GEM
  remote: https://rubygems.org/
  specs:
    actioncable (5.2.3)
      actionpack (= 5.2.3)
      nio4r (~> 2.0)
      websocket-driver (>= 0.6.1)
    actionmailer (5.2.3)
      actionpack (= 5.2.3)
      actionview (= 5.2.3)
      activejob (= 5.2.3)
      mail (~> 2.5, >= 2.5.4)
      rails-dom-testing (~> 2.0)
    actionpack (5.2.3)
      actionview (= 5.2.3)
      activesupport (= 5.2.3)
      rack (~> 2.0)
      rack-test (>= 0.6.3)
      rails-dom-testing (~> 2.0)
      rails-html-sanitizer (~> 1.0, >= 1.0.2)
    actionview (5.2.3)
      activesupport (= 5.2.3)
      builder (~> 3.1)
      erubi (~> 1.4)
      rails-dom-testing (~> 2.0)
      rails-html-sanitizer (~> 1.0, >= 1.0.3)
    activejob (5.2.3)
      activesupport (= 5.2.3)
      globalid (>= 0.3.6)
    activemerchant (1.98.0)
      activesupport (>= 4.2)
      builder (>= 2.1.2, < 4.0.0)
      i18n (>= 0.6.9)
      nokogiri (~> 1.4)
    activemodel (5.2.3)
      activesupport (= 5.2.3)
    activerecord (5.2.3)
      activemodel (= 5.2.3)
      activesupport (= 5.2.3)
      arel (>= 9.0)
    activestorage (5.2.3)
      actionpack (= 5.2.3)
      activerecord (= 5.2.3)
      marcel (~> 0.3.1)
    activesupport (5.2.3)
      concurrent-ruby (~> 1.0, >= 1.0.2)
      i18n (>= 0.7, < 2)
      minitest (~> 5.1)
      tzinfo (~> 1.1)
    acts_as_list (0.9.19)
      activerecord (>= 3.0)
    addressable (2.7.0)
      public_suffix (>= 2.0.2, < 5.0)
    archive-zip (0.12.0)
      io-like (~> 0.3.0)
    arel (9.0.0)
    autoprefixer-rails (9.6.1.1)
      execjs
    awesome_nested_set (3.2.0)
      activerecord (>= 4.0.0, < 7.0)
    bcrypt (3.1.13)
    bindex (0.8.1)
    bootsnap (1.4.5)
      msgpack (~> 1.0)
    builder (3.2.3)
    byebug (11.0.1)
    cancancan (2.3.0)
    canonical-rails (0.2.6)
      rails (>= 4.1, < 6.1)
    capybara (3.29.0)
      addressable
      mini_mime (>= 0.1.3)
      nokogiri (~> 1.8)
      rack (>= 1.6.0)
      rack-test (>= 0.6.3)
      regexp_parser (~> 1.5)
      xpath (~> 3.2)
    capybara-screenshot (1.0.23)
      capybara (>= 1.0, < 4)
      launchy
    carmen (1.1.3)
      activesupport (>= 3.0.0)
    childprocess (2.0.0)
      rake (< 13.0)
    chromedriver-helper (2.1.1)
      archive-zip (~> 0.10)
      nokogiri (~> 1.8)
    climate_control (0.2.0)
    coffee-rails (4.2.2)
      coffee-script (>= 2.2.0)
      railties (>= 4.0.0)
    coffee-script (2.4.1)
      coffee-script-source
      execjs
    coffee-script-source (1.12.2)
    concurrent-ruby (1.1.5)
    crass (1.0.4)
    devise (4.7.1)
      bcrypt (~> 3.0)
      orm_adapter (~> 0.1)
      railties (>= 4.1.0)
      responders
      warden (~> 1.2.3)
    devise-encryptable (0.2.0)
      devise (>= 2.1.0)
    discard (1.1.0)
      activerecord (>= 4.2, < 7)
    erubi (1.8.0)
    execjs (2.7.0)
    ffi (1.11.1)
    font-awesome-rails (4.7.0.5)
      railties (>= 3.2, < 6.1)
    friendly_id (5.2.5)
      activerecord (>= 4.0.0)
    globalid (0.4.2)
      activesupport (>= 4.2.0)
    handlebars_assets (0.23.4)
      execjs (~> 2.0)
      sprockets (>= 2.0.0)
      tilt (>= 1.2)
    i18n (1.6.0)
      concurrent-ruby (~> 1.0)
    io-like (0.3.0)
    jbuilder (2.9.1)
      activesupport (>= 4.2.0)
    jquery-rails (4.3.5)
      rails-dom-testing (>= 1, < 3)
      railties (>= 4.2.0)
      thor (>= 0.14, < 2.0)
    kaminari (1.1.1)
      activesupport (>= 4.1.0)
      kaminari-actionview (= 1.1.1)
      kaminari-activerecord (= 1.1.1)
      kaminari-core (= 1.1.1)
    kaminari-actionview (1.1.1)
      actionview
      kaminari-core (= 1.1.1)
    kaminari-activerecord (1.1.1)
      activerecord
      kaminari-core (= 1.1.1)
    kaminari-core (1.1.1)
    launchy (2.4.3)
      addressable (~> 2.3)
    listen (3.1.5)
      rb-fsevent (~> 0.9, >= 0.9.4)
      rb-inotify (~> 0.9, >= 0.9.7)
      ruby_dep (~> 1.2)
    loofah (2.2.3)
      crass (~> 1.0.2)
      nokogiri (>= 1.5.9)
    mail (2.7.1)
      mini_mime (>= 0.1.1)
    marcel (0.3.3)
      mimemagic (~> 0.3.2)
    method_source (0.9.2)
    mime-types (3.3)
      mime-types-data (~> 3.2015)
    mime-types-data (3.2019.0904)
    mimemagic (0.3.3)
    mini_mime (1.0.2)
    mini_portile2 (2.4.0)
    minitest (5.12.0)
    monetize (1.9.2)
      money (~> 6.12)
    money (6.13.4)
      i18n (>= 0.6.4, <= 2)
    msgpack (1.3.1)
    nio4r (2.5.2)
    nokogiri (1.10.4)
      mini_portile2 (~> 2.4.0)
    orm_adapter (0.5.0)
    paperclip (5.3.0)
      activemodel (>= 4.2.0)
      activesupport (>= 4.2.0)
      mime-types
      mimemagic (~> 0.3.0)
      terrapin (~> 0.6.0)
    paranoia (2.4.2)
      activerecord (>= 4.0, < 6.1)
    polyamorous (2.3.0)
      activerecord (>= 5.0)
    public_suffix (4.0.1)
    puma (3.12.1)
    rack (2.0.7)
    rack-test (1.1.0)
      rack (>= 1.0, < 3)
    rails (5.2.3)
      actioncable (= 5.2.3)
      actionmailer (= 5.2.3)
      actionpack (= 5.2.3)
      actionview (= 5.2.3)
      activejob (= 5.2.3)
      activemodel (= 5.2.3)
      activerecord (= 5.2.3)
      activestorage (= 5.2.3)
      activesupport (= 5.2.3)
      bundler (>= 1.3.0)
      railties (= 5.2.3)
      sprockets-rails (>= 2.0.0)
    rails-dom-testing (2.0.3)
      activesupport (>= 4.2.0)
      nokogiri (>= 1.6)
    rails-html-sanitizer (1.2.0)
      loofah (~> 2.2, >= 2.2.2)
    railties (5.2.3)
      actionpack (= 5.2.3)
      activesupport (= 5.2.3)
      method_source
      rake (>= 0.8.7)
      thor (>= 0.19.0, < 2.0)
    rake (12.3.3)
    ransack (2.3.0)
      actionpack (>= 5.0)
      activerecord (>= 5.0)
      activesupport (>= 5.0)
      i18n
      polyamorous (= 2.3.0)
    rb-fsevent (0.10.3)
    rb-inotify (0.10.0)
      ffi (~> 1.0)
    regexp_parser (1.6.0)
    responders (3.0.0)
      actionpack (>= 5.0)
      railties (>= 5.0)
    ruby_dep (1.5.0)
    rubyzip (1.2.4)
    sass (3.7.4)
      sass-listen (~> 4.0.0)
    sass-listen (4.0.0)
      rb-fsevent (~> 0.9, >= 0.9.4)
      rb-inotify (~> 0.9, >= 0.9.7)
    sass-rails (5.1.0)
      railties (>= 5.2.0)
      sass (~> 3.1)
      sprockets (>= 2.8, < 4.0)
      sprockets-rails (>= 2.0, < 4.0)
      tilt (>= 1.1, < 3)
    sassc (2.2.1)
      ffi (~> 1.9)
    sassc-rails (2.1.2)
      railties (>= 4.0.0)
      sassc (>= 2.0)
      sprockets (> 3.0)
      sprockets-rails
      tilt
    selenium-webdriver (3.142.4)
      childprocess (>= 0.5, < 3.0)
      rubyzip (~> 1.2, >= 1.2.2)
    solidus (2.9.1)
      solidus_api (= 2.9.1)
      solidus_backend (= 2.9.1)
      solidus_core (= 2.9.1)
      solidus_frontend (= 2.9.1)
      solidus_sample (= 2.9.1)
    solidus_api (2.9.1)
      jbuilder (~> 2.8)
      kaminari-activerecord (~> 1.1)
      responders
      solidus_core (= 2.9.1)
    solidus_auth_devise (2.3.0)
      devise (~> 4.1)
      devise-encryptable (= 0.2.0)
      solidus_core (>= 1.2.0, < 3)
      solidus_support (>= 0.1.3)
    solidus_backend (2.9.1)
      autoprefixer-rails
      coffee-rails
      font-awesome-rails (~> 4.0)
      handlebars_assets (~> 0.23)
      jbuilder (~> 2.8)
      jquery-rails
      kaminari (~> 1.1)
      sassc-rails
      solidus_api (= 2.9.1)
      solidus_core (= 2.9.1)
    solidus_core (2.9.1)
      actionmailer (>= 5.1, < 5.3.x)
      actionpack (>= 5.1, < 5.3.x)
      actionview (>= 5.1, < 5.3.x)
      activejob (>= 5.1, < 5.3.x)
      activemerchant (~> 1.66)
      activemodel (>= 5.1, < 5.3.x)
      activerecord (>= 5.1, < 5.3.x)
      activesupport (>= 5.1, < 5.3.x)
      acts_as_list (~> 0.3)
      awesome_nested_set (~> 3.0, >= 3.0.1)
      cancancan (~> 2.2)
      carmen (~> 1.1.0)
      discard (~> 1.0)
      friendly_id (~> 5.0)
      kaminari-activerecord (~> 1.1)
      monetize (~> 1.8)
      paperclip (>= 4.2, < 6)
      paranoia (~> 2.4)
      railties (>= 5.1, < 5.3.x)
      ransack (~> 2.0)
      state_machines-activerecord (~> 0.4)
    solidus_frontend (2.9.1)
      canonical-rails (~> 0.2.0)
      font-awesome-rails (~> 4.0)
      jquery-rails
      kaminari (~> 1.1)
      sassc-rails
      solidus_api (= 2.9.1)
      solidus_core (= 2.9.1)
      truncate_html (~> 0.9, >= 0.9.2)
    solidus_sample (2.9.1)
      solidus_core (= 2.9.1)
    solidus_support (0.3.2)
      capybara-screenshot
    spring (2.1.0)
    spring-watcher-listen (2.0.1)
      listen (>= 2.7, < 4.0)
      spring (>= 1.2, < 3.0)
    sprockets (3.7.2)
      concurrent-ruby (~> 1.0)
      rack (> 1, < 3)
    sprockets-rails (3.2.1)
      actionpack (>= 4.0)
      activesupport (>= 4.0)
      sprockets (>= 3.0.0)
    sqlite3 (1.4.1)
    state_machines (0.5.0)
    state_machines-activemodel (0.7.1)
      activemodel (>= 4.1)
      state_machines (>= 0.5.0)
    state_machines-activerecord (0.6.0)
      activerecord (>= 4.1)
      state_machines-activemodel (>= 0.5.0)
    terrapin (0.6.0)
      climate_control (>= 0.0.3, < 1.0)
    thor (0.20.3)
    thread_safe (0.3.6)
    tilt (2.0.10)
    truncate_html (0.9.3)
    turbolinks (5.2.1)
      turbolinks-source (~> 5.2)
    turbolinks-source (5.2.0)
    tzinfo (1.2.5)
      thread_safe (~> 0.1)
    uglifier (4.1.20)
      execjs (>= 0.3.0, < 3)
    warden (1.2.8)
      rack (>= 2.0.6)
    web-console (3.7.0)
      actionview (>= 5.0)
      activemodel (>= 5.0)
      bindex (>= 0.4.0)
      railties (>= 5.0)
    websocket-driver (0.7.1)
      websocket-extensions (>= 0.1.0)
    websocket-extensions (0.1.4)
    xpath (3.2.0)
      nokogiri (~> 1.8)

PLATFORMS
  ruby

DEPENDENCIES
  bootsnap (>= 1.1.0)
  byebug
  capybara (>= 2.15)
  chromedriver-helper
  coffee-rails (~> 4.2)
  jbuilder (~> 2.5)
  listen (>= 3.0.5, < 3.2)
  puma (~> 3.11)
  rails (~> 5.2.3)
  sass-rails (~> 5.0)
  selenium-webdriver
  solidus (~> 2.9.0)
  solidus_auth_devise
  spring
  spring-watcher-listen (~> 2.0.0)
  sqlite3
  turbolinks (~> 5)
  tzinfo-data
  uglifier (>= 1.3.0)
  web-console (>= 3.3.0)

RUBY VERSION
   ruby 2.6.3p62

BUNDLED WITH
   2.0.1
````

**Comment by kennyadsl** [comment]
@brchristian Thanks for reporting, I think it's already fixed with https://github.com/solidusio/solidus/pull/3320, I'll try to release 2.9.2 as soon as possible.

**Comment by brchristian** [comment]
👍 Sounds good! If we can we get #3320 applied to the `v2.9` branch or in the 2.9.2 release, that would be great. Thank you!

**Comment by kennyadsl** [comment]
For now, I pushed the fix on `v2.9` branch, along with #3331, which is also something worth backporting IMO. I'll try to release 2.9.2 tomorrow. 🙏

**Comment by kennyadsl** [comment]
Released 2.9.2!
